src/app/(landing)/@faq/faq.module.scss

```diff
@@ -19,8 +20,92 @@
     border-radius: 1rem;
     padding: 0.5rem;
   }
+
+  .fish1, .fish2 {
+    animation: fishPhoneMove 15s infinite !important;
+  }
+
+  .fishWaddle {
+    animation: fishWaddle 0.5s infinite !important;
+  }
 }
 
 .item {
   margin-bottom: 0.5rem;
 }
+
+%assets {
+  pointer-events: none;
+  position: absolute;
+  z-index: -1;
+  inset: 0;
+  margin: 0;
+}
+
+.fish1 {
+  @extend %assets;
+  height: auto;
+  width: 4rem;
+  top: 10rem;
+  left: -8rem;
+  animation: fishMove 39.8s infinite;
+}
+
+
+.fish2 {
+  @extend %assets;
+  height: auto;
+  width: 4rem;
+  top: 17rem;
+  left: -7rem;
+  animation: fishMove 39s infinite;
+}
+
+@keyframes fishPhoneMove {
+  0% {
+    transform: translateX(0vw) translate(200%, 250%);
+  }
+  49.99999999999999% {
+    transform: translateX(70vw) translate(200%, 250%);
+  }
+  50% {
+    transform: scaleX(-1) translateX(-70vw) translate(-200%, 250%);
+  }
+  100% {
+    transform: scaleX(-1) translateX(0vw) translate(-200%, 250%);
+  }
+}
+
+@keyframes fishMove {
+  0% {
+    transform: translateX(0vw);
+  }
+  49.99999999999999% {
+    transform: translateX(85vw);
+  }
+  50% {
+    transform: scaleX(-1) translateX(-85vw);
+  }
+  100% {
+    transform: scaleX(-1) translateX(0vw);
+  }
+}
+
+.fishWaddle {
+  @extend %assets;
+  animation: fishWaddle 1s infinite;
+}
+
+@keyframes fishWaddle {
+  0%{
```

Comment:
hmm lets lint this code...

Reply:
fixed! (i think)

Reply:
hmm did u do it manually or run `npm run lint`? because that should just do all the code as a fyi

Reply:
ohhh i just manually did it, i'll npm run lint next time 👍